ddtrace/_trace/trace_handlers.py

```diff
@@ -144,6 +146,24 @@ def _start_span(ctx: core.ExecutionContext, call_trace: bool = True, **kwargs) -
     return span
 
 
+def _finish_span(
```

Comment:
can this be an `ExecutionContext` method instead?

Reply:
no, we want to remove all references to spans from `ExecutionContext`, so I definitely don't want to add anything there